src/pages/index.js

```diff
@@ -131,13 +245,23 @@ function getCardElement(data) {
     imgElement.alt = data.name;
     imgElement.src = data.link;
 
+    if (data.isLiked) {
+        cardLikeBtn.classList.add("card__like-btn_liked");
+    }
+
+
     cardLikeBtn.addEventListener("click", () => {
-        cardLikeBtn.classList.toggle("card__like-btn_liked");
+        if (cardLikeBtn.classList.contains("card__like-btn_liked")) {
+            api.toggleLike(data._id, true)
+                .then(cardLikeBtn.classList.toggle("card__like-btn_liked"));
+        } else {
+            api.toggleLike(data._id, false)
+                .then(cardLikeBtn.classList.toggle("card__like-btn_liked"));
+            console.log(data._id);
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**